src/app/page/MakeFoodBoxMission/Request.style.js

```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+import ImageUpload from "../../component/ImageUpload";
+
+import { makeStyles } from "@material-ui/core/styles";
+
+export const useStyles = makeStyles((theme) => ({
```

Comment:
For future reference we're passing the same theme to the styled components ThemeProvider so you should be able to access the theme there too if you prefer

```js
styled.div`
margin: ${(props) => props.theme.spacing(1)}
`
```